cfgov/paying_for_college/tests/test_scripts.py

```diff
@@ -309,6 +310,32 @@ def test_update_colleges_no_data(self, mock_get_data):
             School.objects.filter(operating=True).exclude(
                 pk=FAKE_SCHOOL_PK).count())
 
+    @unittest.skipUnless(
```

Comment:
Could avoid the duplication here by creating a common test method:

```py
def check_compile_net_prices(self, control):
    mock_api_data = self.mock_results.get('results')[0]
    mock_avg_net_price = mock_api_data.get('latest.cost.avg_net_price.%s' % control.lower())
    school, _ = School.objects.get_or_create(school_id=229115)
    school.control = control
    school = update_colleges.compile_net_prices(school, mock_api_data)
    self.assertEqual(school.avg_net_price, mock_avg_net_price)
    for each in ['0_30k', '30k_48k', '48k_75k', '75k_110k', '110k_plus']:
        self.assertIn(each, school.avg_net_price_slices.keys())

@unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL-dependent')
def test_compile_net_prices_public(self):
    self.check_compile_net_prices('Public')

@unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL-dependent')
def test_compile_net_prices_private(self):
    self.check_compile_net_prices('Private')
```

Reply:
Keepin' it DRY.
√